js/bootstrap-datepicker.js

```diff
@@ -2034,3 +2038,13 @@
 	});
 
 }));
+
+function deprecate(msg) {
+	var console = window.console;
+	if (console && console.warn) {
+		console.warn('DEPRECATED: ' + msg);
```

Comment:
I leave it for now as this whole file is a mixup of spaces vs tabs. But we will clean this up and move to spaces when we move to 2.0